### PR TITLE
External plugins help for cherrypick and refresh plugins

### PR DESCRIPTION
**Pull request by kargakis**

### CODE REVIEW

File: experiment/cherrypicker/server.go [hunk 1]
```diff
@@ -53,6 +54,21 @@ type githubClient interface {
 	ListOrgMembers(org string) ([]github.TeamMember, error)
 }
 
+func HelpProvider(enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: `The cherrypick plugin is used for cherrypicking PRs across branches. For every successful cherrypick invocation a new PR is opened against the target branch and assigned to the requester.`,
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/cherrypick [branch]",
+		Description: "Cherrypick a PR to a different branch.",
```
Comment (cjwagner) [inline experiment/cherrypicker/server.go]: You should mention that this command can be used before or after the PR is merged. That is particularly useful functionality and is worth advertising.

File: experiment/refresh/server.go [hunk 2]
```diff
@@ -32,13 +32,27 @@ import (
...
 
 var refreshRe = regexp.MustCompile(`(?mi)^/refresh\s*$`)
 
+func HelpProvider(enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: `The refresh plugin is used for refreshing status contexts in PRs. Useful in case Github breaks down.`,
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/refresh",
+		Description: "Refresh status contexts in PRs",
```
Comment (cjwagner) [inline experiment/refresh/server.go]: s/in PRs/on a PR./

**Comment by kargakis** [comment]
comments addressed

/hold cancel

**Pull request merged**